src/AutoRest.CSharp.V3/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -18,6 +18,7 @@ internal class CSharpProj : IPlugin
     <TargetFramework>netstandard2.0</TargetFramework>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Nullable>annotations</Nullable>
+    <NoWarn>CS1591</NoWarn>
```

Comment:
Don't add it here, instead include it into a shared props file https://github.com/Azure/autorest.csharp/blob/feature/v3/test/TestServerProjects/Directory.Build.props

Reply:
Done.